### PR TITLE
Adding kitematic to the readme

### PR DESCRIPTION
**Pull request by spf13**

### CODE REVIEW

**Comment by jfrazelle** [comment]
cool gordon

**Comment by moxiegirl** [comment]
LGTM

**Comment by SvenDowideit** [comment]
LGTM

**Comment by moxiegirl** [comment]
@jfrazelle is this ok to merge --- builds r failing

**Comment by jfrazelle** [comment]
Needs sig

**Comment by jfrazelle** [comment]
We were testing the DCO labels but still needs a sig;)

**Comment by SvenDowideit** [comment]
@spf13 it seems you need to sign your contribution?

File: README.md [hunk 1]
```diff
@@ -243,4 +240,8 @@ for a container-centric world
 system 
 * [Docker Compose](https://github.com/docker/compose) (formerly Fig): 
 Define and run multi-container apps
+* [Kitematic](https://github.com/kitematic/kitematic): The easiest way to use Docker on a Mac
```
Comment (ahmetalpbalkan) [inline README.md]: add docker/distribution while you're at it maybe since you'll amend for DCO anyway?

**Pull request closed without merging**